### PR TITLE
Implement a draw method in scenario config for debug purpose. 

### PR DESCRIPTION
**Pull request by jrobinAV**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [x] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description

A draw method has been implemented to export a scenario config graph as a PNG file.
The method requires `matplotlib` to be installed to run. However, `matplotlib` should be installed independently from taipy as it is not a taipy dependency.

## Related Tickets & Documents

- Closes Issue #1592 

## Checklist
_We encourage you to keep the code coverage percentage at 80% and above._

- [x] Does this solution meet the acceptance criteria of the related issue?
- [x] Is the related issue checklist completed?
- [x] Does this PR add unit tests for the developed code? The unit tests are skipped as they contain no assert.
- [x] End-to-End tests have been added or updated? Not applicable
- [x] Was the documentation updated, or was a dedicated issue for documentation created? (If applicable)
- [x] Have the release notes been updated? (If applicable)

### CODE REVIEW

File: taipy/core/config/scenario_config.py [hunk 1]
```diff
@@ -373,3 +372,64 @@ def _set_default_configuration(
...
+                               nodelist=[node for node in graph.nodes if isinstance(node, TaskConfig)],
+                               node_color="orange",
+                               node_shape="D",
+                               node_size=2000)
+        nx.draw_networkx_labels(graph, pos, labels=labls)
+        nx.draw_networkx_edges(graph, pos, node_size=2000, edge_color="black", arrowstyle="->", arrowsize=25)
+        path = file_path or f"{self.id}.png"
+        plt.savefig(path)
+        plt.close()  # Close the plot to avoid display
+        logger.info(f"Graph exported as image: {path}")
```
Comment (trgiangdo) [inline taipy/core/config/scenario_config.py]:
```suggestion
        logger.info(f"The graph image of the scenario configuration `{self.id}` is exported: {path}")
```

File: tests/core/config/test_scenario_config.py [hunk 2]
```diff
@@ -299,3 +301,80 @@ def test_add_sequence():
...
+    dn_config_5 = Config.configure_data_node("dn5")
+    task_config_1 = Config.configure_task("task1", sum, input=[dn_config_1, dn_config_2], output=dn_config_3)
+    task_config_2 = Config.configure_task("task2", sum, input=[dn_config_1, dn_config_3], output=dn_config_4)
+    task_config_3 = Config.configure_task("task3", print, input=dn_config_4)
+    scenario_cfg = Config.configure_scenario(
+        "scenario1",
+        [task_config_1, task_config_2, task_config_3],
+        [dn_config_5],
+    )
+    scenario_cfg.draw()
```
Comment (trgiangdo) [inline tests/core/config/test_scenario_config.py:320]: We can assert the path exists when running this test and similarly for other draw tests

**Review by trgiangdo — DISMISSED** [review]
LGTM

File: tests/core/config/test_scenario_config.py (hunk 2 above)
Comment (jrobinAV) [reply tests/core/config/test_scenario_config.py:320]: matplotlib won't be installed in GitHub. So it should do nothing. I don't know how to test that, then.

**Pull request merged**